src/models/NFTAttributes.ts

```diff
@@ -8,7 +8,9 @@ export enum ERCType {
 
 export enum NeverminedNFT721Type {
     nft721 = 'nft721', // Standard 721 implementation
-    nft721Subscription = 'nft721-subscription' // 721 implementing subscriptions that can expire
+    nft721Subscription = 'nft721-subscription', // 721 implementing subscriptions that can expire
+    nft721POAP = 'nft721-poap', // 721 implementing a Proof of Attendance NFT
+    nft721SoulBound = 'nft721-soulbound' // 721 implementing a Proof of Attendance NFT
```

Comment:
is this being used?

Reply:
It will be as soon as we document a poap or soulbound flow